addon/Config.jsm

```diff
@@ -37,31 +37,7 @@ var config = {
      * - If there is no key for an endStudy reason, no url will open.
      * - usually surveys, orientations, explanations
      */
-    endings: {
```

Comment:
We're deploying this in stealth-mode, so there's no need to open tabs for different study endings.